### DISCUSSION TITLE
can yall add WVPB plz?

### DISCUSSION THREAD

**Discussion by jayc3-cmyk** [iptv-org/community#577] [Q&A] [answered]
i have a livestream link if needed

**Comment by kaikyoofers** [comment]
@jayc3-cmyk can you share the livestream link here?

**Reply by jayc3-cmyk** [reply]
Okay, i am back. Will this work? If not, then contact at adrieneriksen6@gmail.com 
https://player.pbs.org/ga-livestream-stationplayer/?station_id=f36a8eaa-cc11-4e0d-a3c1-638b9ebd3023&unsafePostMessages=true&feed_cid=2f521f3c-746d-469b-9d77-572900603a99

**Reply by kaikyoofers** [reply, marked as answer]
> Okay, i am back. Will this work? If not, then contact at adrieneriksen6@gmail.com 
> https://player.pbs.org/ga-livestream-stationplayer/?station_id=f36a8eaa-cc11-4e0d-a3c1-638b9ebd3023&unsafePostMessages=true&feed_cid=2f521f3c-746d-469b-9d77-572900603a99

Does not work "It looks like you’re outside of the WVPB broadcast area."

**Reply by jayc3-cmyk** [reply]
Here's an image if you wanna put an image for the channel preview https://th.bing.com/th/id/R.d3e8f5ce7361edff790be0a70cc05006?rik=d9ibjFVA6UYH4Q&riu=http%3a%2f%2fmediad.publicbroadcasting.net%2fp%2fwvpn%2ffiles%2fstyles%2fx_large%2fpublic%2f201510%2fwvpb_logo_-_square.png&ehk=Bf8JeOupPdJI33Yjrim2qgP1%2b7alxE5BzubrJbCOHlw%3d&risl=&pid=ImgRaw&r=0

Still tryna find it

**Comment by jayc3-cmyk** [comment]
Yeah, hold on. I'm in a supermarket right now and I don’t have my laptop on
me.
